src/www/ui/user-edit.php

```diff
@@ -55,6 +55,14 @@ protected function handle(Request $request)
     $SessionUserRec = $this->GetUserRec($user_pk);
     $SessionIsAdmin = $this->IsSessionAdmin($SessionUserRec);
 
+    if (!($SessionIsAdmin or
```

Comment:
Since `or` is evaluated lazy, $user_pk_to_modify is not definined for $SessionIsAdmin==true, but required to compute !(true or anything) or $user_pk == $user_pk_to_modify in that case.